src/frequenz/sdk/actor/power_distributing/_distribution_algorithm/_distribution_algorithm.py

```diff
@@ -752,17 +781,21 @@ def _inclusion_exclusion_bounds(
         excl_bounds: dict[int, float] = {}
         for battery, inverters in components:
             if supply:
-                excl_bounds[battery.component_id] = -battery.power_exclusion_lower_bound
-                incl_bounds[battery.component_id] = -battery.power_inclusion_lower_bound
+                excl_bounds[
+                    battery.component_id
+                ] = -battery.power_bounds.exclusion_lower
+                incl_bounds[
+                    battery.component_id
+                ] = -battery.power_bounds.inclusion_lower
```

Comment:
the assignment of `excl_bounds` and `incl_bounds`  should probably fit in 2 lines as it was done on the `else` branch

Reply:
I believe this was done by `black`

Reply:
I see, these lines are one character longer than the one on the else-branch due to the sign `-`. I thought they were the same length.